### ISSUE TITLE
Use github templates for issues and pull requests

### ISSUE THREAD

**Issue by kddrake** [Jropp/jason-ropp-js-iobp#41]
## Main Goal
Add [templates](https://blog.github.com/2016-02-17-issue-and-pull-request-templates/) for issues and PRs to your repo so that it's easier and quicker to fill out either.

### When Accomplished
- [x] When opening an issue, headers will already be added and include descriptions/examples
- [x] When opening a PR, headers will already be added and include descriptions/examples

## Notes
- Use the [BO PR template](https://github.com/Banno/platform-ux/blob/master/.github/PULL_REQUEST_TEMPLATE.md)
- Will use (for now) this custom template for issues:
```
# Please complete these issues before attempting this issue
- [ ] _Include linked issue numbers that need to be finished before this issue is started (doesn't have to be a dependency)_

## Main Goal
_Describe generally what will be added/changed_

### When Accomplished
- [ ] _Specific items that need to work or be observed before the issue is completed_

## Notes
_Include that otherwise needs to be said. Good place for hints or resources._

### Dependencies
- [ ] _Linked issues or PRs required to be complete before work on this issue is possible_ 
```

**Comment by kddrake** [comment]
Make sure to add a template for issues as well

**Comment by Jropp** [comment]
@kddrake on the PRs I removed the 'create tests' checkboxes. Are we going to be doing any testing on this project? Should I add that back in?

**Comment by kddrake** [comment]
@Jropp we should... but we'll see. you're good to leave it out for the time being (we can add it back in later if need be).

**Comment by kddrake** [comment]
closed by 488b80f and c846e66
